File: java_work/ex09/src/ex01.java
// 클래스 정의
/* main은 메모리에 무조건 존재한다.
*
*
*/

class A{
    int num = 10;
    public void doA(){
        System.out.println("아무거나 만들기");
    }
}

//목록의 이름이 초록색인 것은 깃허브에 업로드 안한 파일이다.

public class ex01 {
    public static void main(String[] args) {

        int num = 10;
        int num2 = 20;
        int num3 = 30;

//        선언하는 순간 메모리에 적재된다.
        A a1 = new A();
        A a2 = new A();
        A a3 = new A();
//        a1, a2, a3 는 참조변수
//        new는 객채열기.

        System.out.println(a1.num);
        a1.doA();
        a1. num = 30;
        System.out.println("a1.num = "+a1.num);

        System.out.println("a2.num = "+a2.num);
        a2.doA();
    }
}

File: html_work/15/ex01.js
// let 변경 가능한 변수 
// const는 변경 불가능한 상수
const aa = document.getElementById("btn-min");
let bb = document.getElementById("btn-max");
const printDiv = document.getElementById("print-div");

const n1 = document.getElementById("num1");
const n2 = document.getElementById("num2");
const n3 = document.getElementById("num3");
const n4 = document.getElementById("num4");
const n5 = document.getElementById("num5");

function min() {
  const arr = [n1.value, n2.value, n3.value, n4.value, n5.value];
  let min = n1.value;
  for (let index = 0; index < arr.length; index++) {
    const element = arr[index];
    if (Number(min) > Number(element))
      min = element;
  }
  printDiv.textContent = "최소값 = " + min;
}

function max() {
  const arr = [n1.value, n2.value, n3.value, n4.value, n5.value];
  let max = arr[0];
  for (const key of arr) {
    if (Number(max) < Number(key))
      max = key;
  }
  printDiv.textContent = "최대값 = " + max;
}

aa.addEventListener('click', min);
bb.addEventListener('click', max);

printDiv.textContent = "안녕하세요 내용을 넣는 거에요."


dd = 30;
console.log(dd);
for (let i = 0; i < 10; i++) {
  console.log(i);
}
